index.html

```diff
@@ -6,43 +6,178 @@
-      href="https://fonts.googleapis.com/css?family=Crete Round|Roboto|Poppins|"
+      href="https://fonts.googleapis.com/css?family=Crete Round|Roboto|Poppins|Inter|InaiMathi|"
     />
   </head>
   <body>
-    <header>
-      <div class="toolbar">
-        <span><a href="">John Doe</a></span>
-        <img src="./assets//menu.png" />
-      </div>
-    </header>
-    <section class="headline">
-      <span class="title">Hey there. <br />I'm Alex</span>
-      <span class="sub-title">I'am a software Developer</span>
-      <p>
-        I can help you build a product , feature or website Look through some of
-        my work and experience! If you like what you see and have a project you
-        need coded, don’t hestiate to contact me.
-      </p>
-      <div class="socials">
-        <ul>
-          <li>
-            <a href=""><img src="./assets/github.png" alt="github" /></a>
-          </li>
-          <li>
-            <a href=""> <img src="./assets/linkedin.png" alt="linkedin" /></a>
-          </li>
-          <li>
-            <a href=""><img src="./assets//Vector.png" alt="vector1" /></a>
-          </li>
-          <li>
-            <a href=""><img src="./assets//twitter.png" alt="twitter" /></a>
-          </li>
-          <li>
-            <a href=""><img src="./assets//Vector2.png " alt="vector2" /></a>
-          </li>
-        </ul>
-      </div>
-    </section>
+    <main>
+      <header>
+        <div class="toolbar">
+          <span><a href="">John Doe</a></span>
+          <img src="./assets//menu.png" />
+        </div>
+      </header>
+      <section class="headline">
+        <span class="title">Hey there. <br />I'm Alex</span>
+        <span class="sub-title">I'am a software Developer</span>
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <div class="socials">
+          <ul>
+            <li>
+              <a href=""><img src="./assets/github.png" alt="github" /></a>
+            </li>
+            <li>
+              <a href=""> <img src="./assets/linkedin.png" alt="linkedin" /></a>
+            </li>
+            <li>
+              <a href=""><img src="./assets//Vector.png" alt="vector1" /></a>
+            </li>
+            <li>
+              <a href=""><img src="./assets//twitter.png" alt="twitter" /></a>
+            </li>
+            <li>
+              <a href=""><img src="./assets//Vector2.png " alt="vector2" /></a>
+            </li>
+          </ul>
+        </div>
+      </section>
+      <section class="works">
+        <div class="title">
+          <span>My Recent Works</span>
+          <hr />
+        </div>
+        <div class="work-container">
+          <div class="hero-work">
+            <img src="./assets//hero-work.png" alt="hero" />
+            <span>Multi-Post Stories</span>
```

Comment:
- [ ] [REQUIRED] Please replace the `span` in this `<span>Multi-Post Stories</span>` to a header tag, either of them `h2`, `h3`, `h4` or `h5`. Please refer to my previous comment for the reason behind this suggestion.